### PR TITLE
Phishing campaign widget

### PR DESCRIPTION
**Pull request by erezh31**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by erezh31** [comment]
@yguezpa pleae review the script
@idovandijk please review the rest - widgets, dashboards, etc

File: Packs/Campaign/Scripts/DBotGroupPhishingIncidents/DBotGroupPhishingIncidents.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from CommonServerPython import *
+
+model_name = 'phishing_clustering'
+field_for_grouping = 'emailsubject,emailbody,emailbodyhtml'
```
Comment (yguezpa) [inline Packs/Campaign/Scripts/DBotGroupPhishingIncidents/DBotGroupPhishingIncidents.py:4]: Why taking **emailbody** and **emailbodyhtml** ?  What the difference between them ? Won't it bring redundancy ?

**Review by yguezpa — APPROVED** [review]

File: Packs/Campaign/Scripts/DBotGroupPhishingIncidents/DBotGroupPhishingIncidents.py (hunk 1 above)
Comment (erezh31) [reply Packs/Campaign/Scripts/DBotGroupPhishingIncidents/DBotGroupPhishingIncidents.py:4]: usually only one of them exists but I don't know which one of them, so better to show both

File: Packs/Campaign/Scripts/DBotGroupPhishingIncidents/DBotGroupPhishingIncidents.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+name: DBotGroupPhishingIncidents
+script: '-'
+subtype: python3
+system: false
+tags:
+- ml
+timeout: '0'
+type: python
+fromversion: 6.2.0
+dockerimage: demisto/python3:3.9.6.22912
```
Comment (yaakovi) [inline Packs/Campaign/Scripts/DBotGroupPhishingIncidents/DBotGroupPhishingIncidents.yml:92]:
```suggestion
dockerimage: demisto/python3:3.9.6.22912
tests:
- No test
```

**Comment by erezh31** [comment]
image
Filter in:
image

**Comment by erezh31** [comment]
@idovandijk please review the widgets \ dashboards

**Comment by erezh31** [comment]
@ShirleyDenkberg please review the description

**Comment by ShirleyDenkberg** [comment]
@yaakovi @idobloch @yguezpa Doc review completed.

**Review by idovandijk — APPROVED** [review]

**Comment by erezh31** [comment]
@yaakovi I think we are good to go - no playbook test, can you please merge?

**Comment by Itay4** [comment]
what is the status here ?

**Pull request closed without merging**